app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -227,6 +227,12 @@ public function indexAction($page = 1)
         }
         $session->set('mautic.email.page', $page);
 
+        /** @var \Mautic\PageBundle\Model\PageModel $model */
+        $pageModel = $this->getModel('page.page');
+
+        /** @var \Mautic\PageBundle\Entity\HitRepository $pageRepo */
+        $pageRepo = $pageModel->getHitRepository();
```

Comment:
It's actually `HitRepository`, so the `$pageRepo` variable name can be confusing.

Reply:
Done